### PR TITLE
Voxkrynn;vulpinesystems.json

### PR DESCRIPTION
**Pull request by VoxKrynn**

### CODE REVIEW

**Review by TheGiddyLimit — CHANGES_REQUESTED** [review]
file should be moved to e.g. `creature/VoxKrynn; Vulpine Systems.json` (note the addition of spaces, too)

File: creature/VoxKrynn; Vulpine Systems.json [hunk 1]
```diff
@@ -0,0 +1,956 @@
...
+			{
+				"json": "VulpineSystems",
+				"abbreviation": "VS",
+				"full": "Vulpine Systems",
+				"url": "",
+				"authors": [
+					"Vox Krynn"
+				],
+				"convertedBy": [],
+				"color": "#490085"
```
Comment (TheGiddyLimit) [inline creature/VoxKrynn; Vulpine Systems.json:13]:
```suggestion
				"color": "490085"
```

File: creature/VoxKrynn; Vulpine Systems.json [hunk 2]
```diff
@@ -0,0 +1,956 @@
+{
+	"_meta": {
+		"sources": [
+			{
+				"json": "VulpineSystems",
+				"abbreviation": "VS",
+				"full": "Vulpine Systems",
+				"url": "",
```
Comment (TheGiddyLimit) [inline creature/VoxKrynn; Vulpine Systems.json:8]:
```suggestion
```

File: creature/VoxKrynn; Vulpine Systems.json [hunk 3]
```diff
@@ -0,0 +1,956 @@
...
+				"authors": [
+					"Vox Krynn"
+				],
+				"convertedBy": [],
```
Comment (TheGiddyLimit) [inline creature/VoxKrynn; Vulpine Systems.json:12]:
```suggestion
```

File: creature/VoxKrynn; Vulpine Systems.json [hunk 4]
```diff
@@ -0,0 +1,954 @@
...
+				"authors": [
+					"Vox Krynn"
+				],
+				"color": "490085"
+			}
+		]
+	},
+	"monster": [
+		{
+			"uniqueId": "1b04ebf4-2ac4-432c-9721-dd8b2ec5b637",
```
Comment (TheGiddyLimit) [inline creature/VoxKrynn; Vulpine Systems.json:17]:
```suggestion
```

File: creature/VoxKrynn; Vulpine Systems.json [hunk 5]
```diff
@@ -0,0 +1,954 @@
...
+						"Golden Beetle has advantage on Dexterity (Stealth) checks made to hide in a gold vein, or among five other Golden Beetles."
+					]
+				}
+			],
+			"damageTags": [
+				"P"
+			]
+		},
+		{
+			"uniqueId": "d6698f05-3463-419d-965e-99ab8e657c69",
```
Comment (TheGiddyLimit) [inline creature/VoxKrynn; Vulpine Systems.json:116]:
```suggestion
```

File: creature/VoxKrynn; Vulpine Systems.json [hunk 6]
```diff
@@ -0,0 +1,954 @@
...
+			"environment": [
+				"mountain",
+				"underdark"
+			],
+			"damageTags": [
+				"P"
+			]
+		},
+		{
+			"uniqueId": "a9f8fa42-bc66-460c-9dc7-5661c96091c6",
```
Comment (TheGiddyLimit) [inline creature/VoxKrynn; Vulpine Systems.json:215]:
```suggestion
```

File: creature/VoxKrynn; Vulpine Systems.json [hunk 7]
```diff
@@ -0,0 +1,954 @@
...
+			"tokenHref": {
+				"type": "external",
+				"url": "https://cdn.discordapp.com/attachments/710988583108083774/1217031093979447366/V-CrystalWasp.png?ex=66028b88&is=65f01688&hm=c1c6375e340909ee84e0e51995610b1c1ed1caf0c4c0d9f817a2ce59ff2e6cfa&"
+			},
+			"damageTags": [
+				"P"
+			]
+		},
+		{
+			"uniqueId": "b60e91b5-92b3-47dc-ac7a-7608ef070ccf",
```
Comment (TheGiddyLimit) [inline creature/VoxKrynn; Vulpine Systems.json:301]:
```suggestion
```

File: creature/VoxKrynn; Vulpine Systems.json [hunk 8]
```diff
@@ -0,0 +1,954 @@
...
+			"tokenHref": {
+				"type": "external",
+				"url": "https://cdn.discordapp.com/attachments/710988583108083774/1217042422497017876/V-CrystalMoss.png?ex=66029615&is=65f02115&hm=2f843baa3bc94f0c6af08b32973832357d6cbe06e22f57ceaf10d68127f59469&"
+			},
+			"damageTags": [
+				"N"
+			]
+		},
+		{
+			"uniqueId": "86145d67-e970-4c72-9daa-a09ff4e62d21",
```
Comment (TheGiddyLimit) [inline creature/VoxKrynn; Vulpine Systems.json:376]:
```suggestion
```

File: creature/VoxKrynn; Vulpine Systems.json [hunk 9]
```diff
@@ -0,0 +1,954 @@
...
+			"tokenHref": {
+				"type": "external",
+				"url": "https://cdn.discordapp.com/attachments/710988583108083774/1218056807562608770/V-VeiledBandit1.png?ex=660646cd&is=65f3d1cd&hm=8b1736dbca18abb098433e1697b09cd967482840ad29a4c09c6be28c77592c73&"
+			},
+			"damageTags": [
+				"P"
+			]
+		},
+		{
+			"uniqueId": "eacb47de-5e95-4185-9937-0905a6468ea0",
```
Comment (TheGiddyLimit) [inline creature/VoxKrynn; Vulpine Systems.json:498]:
```suggestion
```

File: creature/VoxKrynn; Vulpine Systems.json [hunk 10]
```diff
@@ -0,0 +1,954 @@
...
+			"tokenHref": {
+				"type": "external",
+				"url": "https://cdn.discordapp.com/attachments/710988583108083774/1218056816639086592/V-VeiledBanditCaptian.png?ex=660646cf&is=65f3d1cf&hm=a41b20d6d22b5362d009ef7e243a823bcccaf23126ee8b9c26fe9d0655728d31&"
+			},
+			"damageTags": [
+				"P"
+			]
+		},
+		{
+			"uniqueId": "7816c494-2568-408e-88db-34ff8b7f67d7",
```
Comment (TheGiddyLimit) [inline creature/VoxKrynn; Vulpine Systems.json:625]:
```suggestion
```

File: creature/VoxKrynn; Vulpine Systems.json [hunk 11]
```diff
@@ -0,0 +1,954 @@
...
+			],
+			"actionTags": [
+				"Multiattack"
+			],
+			"damageTags": [
+				"P"
+			]
+		},
+		{
+			"uniqueId": "75b36ff7-f31a-4152-8eef-cc9f76197d52",
```
Comment (TheGiddyLimit) [inline creature/VoxKrynn; Vulpine Systems.json:763]:
```suggestion
```

**Review by TheGiddyLimit — CHANGES_REQUESTED** [review]

**Comment by TheGiddyLimit** [comment]
@VoxKrynn apologies, I meant to follow up on this

do you no longer want the file added to the repo? It probably only needs a couple more minor fixes; I _think_ they're all from bugs in the homebrew builder (hopefully fixed for next release), hence my meddling

the `alignment` issues I thought I already fixed, which version of 5etools are you using? I.e. are you still on the old https://5etools-mirror-1.github.io ver..?

**Comment by VoxKrynn** [comment]
Sorry I just don't really know what I am doing here, and I realized that I will have to keep doing this when I add more things to the homebrew. So I think it would just be easier for me to share it manually. For the version I am using I am unsure, it is just the standard version that you get when you just google it. Just looked before sending this, I am using 1.201.1 5eTools.

**Comment by TheGiddyLimit** [comment]
fair enough, no worries

**Pull request closed without merging**